src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.test.tsx

```diff
@@ -156,95 +171,83 @@ describe("MeasureInformation component", () => {
   });
 
   it("Should display a delete button if user is the owner of measure", async () => {
-    await act(async () => {
-      const { findByTestId } = render(<MeasureInformation />);
-      const result: HTMLElement = await findByTestId("delete-measure-button");
-      expect(result).toBeInTheDocument();
-    });
+    render(<MeasureInformation />);
+    const result: HTMLElement = await findByTestId("delete-measure-button");
+    expect(result).toBeInTheDocument();
   });
 
-  it("Should not display a delete button if user is not the owner of measure", async () => {
-    useOktaTokensMock.mockImplementation(() => ({
-      getUserName: () => "othertestuser@example.com",
+  it("Should not display a delete button if user is not the owner of measure", () => {
+    useOktaTokens.mockImplementationOnce(() => ({
+      getUserName: () => "othertestuser@example.com", //#nosec
     }));
-    await act(async () => {
-      const { queryByText } = render(<MeasureInformation />);
-      const result: HTMLElement = queryByText("delete-measure-button");
-      expect(result).toBeNull();
-    });
+    render(<MeasureInformation />);
+    const result: HTMLElement = queryByText("delete-measure-button");
+    expect(result).toBeNull();
   });
 
   it("On delete click, user is presented with a confirm deletion screen", async () => {
-    await act(async () => {
-      const { findByTestId, getByTestId } = await render(
-        <MeasureInformation />
-      );
-      const result: HTMLElement = await findByTestId("delete-measure-button");
-      fireEvent.click(result);
-      const confirmDelete = await getByTestId("delete-measure-button-2");
-      expect(confirmDelete).toBeInTheDocument();
-      const cancelDelete = await getByTestId("cancel-delete-measure-button");
-      expect(cancelDelete).toBeInTheDocument();
-    });
+    render(<MeasureInformation />);
+    const result: HTMLElement = await findByTestId("delete-measure-button");
+    fireEvent.click(result);
+    const confirmDelete = await getByTestId("delete-measure-button-2");
```

Comment:
Intellij says the await here is unecessary:
>TS80007: 'await' has no effect on the type of this expression.

Annoyingly, it's not a warning or error and only appears when I hover over the statement :/